src/AutoMapper/FeatureCollection.cs

```diff
@@ -0,0 +1,16 @@
+﻿using AutoMapper.Internal;
+
+namespace AutoMapper
+{
+    public class FeatureCollection : FeatureCollectionBase<IFeature>, IFeature
+    {
+        void IFeature.Seal(IConfigurationProvider configurationProvider)
```

Comment:
This can be public and then the casts below are not needed anymore.

Reply:
I was making it explicit implementation to not show the method to the outside, it is only the internal api that should call this method. 

Making the method `internal` will work for the use-case also, but then it will be harder to write unit-tests for it (if we not allowing internal members for the unit tests).

Should it be hidden for the normal user with either the explicit implementation or internal?

Reply:
Our convention right now is to internalize it.